app/controllers/YourContactDetailsController.scala

```diff
@@ -50,23 +75,20 @@ class YourContactDetailsController @Inject()(authenticate: IdentifierAction,
 
       companyName <- dataStoreService.getOwnCompanyName(request.user.eori)
       dataStoreAddress <- dataStoreService.getCompanyAddress(request.user.eori)
-
       companyAddress: CompanyAddress = dataStoreAddress.getOrElse(
         new CompanyAddress("","",Some(""),""))
 
       address = CompanyAddress(
         streetAndNumber = companyAddress.streetAndNumber,
         city = companyAddress.city,
         postalCode = companyAddress.postalCode,
-        countryCode = companyAddress.countryCode
-      )
-
-      sessionId = hc.sessionId.getOrElse({log.error("Missing SessionID"); SessionId("Missing Session ID")})
-      accountlinks <- sessionCacheConnector.getAccontLinks(sessionId.value)
+        countryCode = companyAddress.countryCode)
 
+      accountlinks <- sessionCacheConnector.getAccontLinks(localSessionId.value)
     } yield {
       Ok(view(request.user.eori, accountlinks.getOrElse(Seq.empty[AccountLinkWithoutDate]),
-        companyName, address, email.toString))
+        companyName, address, email.toString)(request, messages, appConfig))
     }
   }
 }
+
```

Comment:
Please remove this extra line break. There has to be only one line break at the end of the file